tests/test_wasapi_client.py

```diff
@@ -287,34 +454,28 @@ class TestDownloader:
                       'md5': '62f87a969af0dd857ecd6c3e7fde6aed'}
     }
 
-    def test_run(self, mock_download, mock_session):
+    def test_run(self, mock_download):
         """Test downloader when downloads are successful."""
-        msg = '{}: 200 OK'.format(self.FILE_DATA['locations'][0])
-        mock_download.return_value = msg
         # Create a queue holding two sets of file data.
         get_q = multiprocessing.JoinableQueue()
         for _ in (1, 2):
             get_q.put(self.FILE_DATA)
         result_q = multiprocessing.Queue()
         log_q = multiprocessing.Queue()
-
-        p = wc.Downloader(get_q, result_q, log_q)
-        p.start()
-        p.run()
-        p.join()
+        with patch('wasapi_client.verify_file', return_value=True):
+            p = wc.Downloader(get_q, result_q, log_q)
+            p.start()
+            p.run()
```

Comment:
Does p.run() actually need to be called? Isn't it enough to just call p.start()?

Reply:
No it doesn't need to be called explicitly.